hack/generate_addons.sh

```diff
@@ -19,10 +20,11 @@ kustomize_build() {
 
 helm_template eks aws-node-termination-handler 0.13.3
 helm_template autoscaler cluster-autoscaler 1.0.4 -chart
-helm_template nvdp nvidia-device-plugin 0.8.2
+helm_template nvdp nvidia-device-plugin 0.7.0
+helm_template eks aws-load-balancer-controller 1.1.0
 
 curl -o $ADDONS_DIR/kustomize/overlays/metrics-server/resources/metrics-server.yaml -L https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.3.7/components.yaml
 kustomize_build metrics-server
 curl -o $ADDONS_DIR/kustomize/overlays/aws-ebs-csi-driver/resources/crd_snapshotter.yaml -L https://raw.githubusercontent.com/kubernetes-sigs/aws-ebs-csi-driver/v0.7.0/deploy/kubernetes/cluster/crd_snapshotter.yaml
 kustomize_build aws-ebs-csi-driver
-kustomize_build aws-alb-ingress-controller
+kustomize_build aws-load-balancer-controller-crds
```

Comment:
We're getting the manifests from the helm chart, but the CRDs via a kustomize base in the repo repository. Can we not just references the bases from https://github.com/kubernetes-sigs/aws-load-balancer-controller/tree/main/config

Reply:
yeah that makes sense